lib/controllers/frontend/spree/user_confirmations_controller.rb

```diff
@@ -11,4 +11,9 @@ class Spree::UserConfirmationsController < Devise::ConfirmationsController
   def after_confirmation_path_for(resource_name, resource)
     signed_in?(resource_name) ? signed_in_root_path(resource) : spree.login_path
   end
+
+  def translation_scope
+    'devise.confirmations'
```

Comment:
It seems not `devise.user_confirmations` but `devise.confirmations`.
https://github.com/spree/spree_auth_devise/blob/master/config/locales/en.yml#L7